legacy/schemas/com.wire.android.core.storage.db.global.GlobalDatabase/1.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Why are we adding this to the legacy module??? Remove please

Reply:
no idea actually O.o